features/preselecting_refund_payment_method_based_on_order.feature

```diff
@@ -0,0 +1,27 @@
+@refunds
+Feature: Refunding all order units
+    In order to give back money for all of the bought products to Customer
+    As an Administrator
+    I want to be able to refund all order units
+
+    Background:
+        Given the store operates on a single green channel in "United States"
+        And the store has a product "Mr. Meeseeks T-Shirt" priced at "$10.00"
+        And the store has "Galaxy Post" shipping method with "$10.00" fee
+        And the store allows paying with "Space money"
+        And there is a customer "rick.sanchez@wubba-lubba-dub-dub.com" that placed an order "#00000022"
+        And the customer bought 3 "Mr. Meeseeks T-Shirt" products
+        And the customer chose "Galaxy Post" shipping method to "United States" with "Space money" payment
+        And I am logged in as an administrator
+        And the order "#00000022" is already paid
+
+    @ui @javascript
+    Scenario: Refunding all order units
+        When I want to refund some units of order "#00000022"
+        And the selected refund payment method should be "Space money"
```

Comment:
`Then the selected refund payment method should be "Space money"` and the rest of steps are not required :)